pkg/metrics/producers/scheduledcapacity/gauges.go

```diff
@@ -0,0 +1,25 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scheduledcapacity
+
+import "github.com/awslabs/karpenter/pkg/metrics"
+
+const (
+	Subsystem = "scheduled_capacity"
```

Comment:
Does `Subsystem` needs to be an exported field, can it be local to the package?

Reply:
It's used twice. Once in `gauges.go` and once in `producer.go`.